tests/unit/Serializers/ClaimSerializerTest.php

```diff
@@ -190,7 +190,7 @@ public function serializationProvider() {
 
 		$claim = new Statement( new Claim( new PropertyNoValueSnak( 42 ) ) );
 		$claim->setReferences( new ReferenceList( array(
-			new Reference()
+			new Reference( array( new PropertyNoValueSnak( 1 ) ) )
```

Comment:
This test case does not make sense any more. The `references > snaks` array below is empty. How is that possible?